spotlight-client/src/RacialDisparitiesNarrativePage/RacialDisparitiesNarrativePage.tsx

```diff
@@ -93,59 +97,51 @@ const RacialDisparitiesNarrativePage: React.FC<RacialDisparitiesNarrativePagePro
   const templateData = wrapDynamicText(narrative.templateData);
 
   return (
-    <NarrativeLayout
-      sections={[
-        {
-          title: narrative.title,
-          contents: (
-            <NarrativeIntroContainer>
-              <NarrativeTitle>{narrative.title}</NarrativeTitle>
-              {narrative.isLoading || narrative.isLoading === undefined ? (
-                <Loading />
-              ) : (
-                <IntroCopy>
-                  {HTMLReactParser(pupa(narrative.introduction, templateData))}
-                </IntroCopy>
-              )}
-              {narrative.populationDataSeries && (
-                <BarChartPair
-                  data={narrative.populationDataSeries}
-                  download={() => narrative.downloadPopulation()}
-                  filters={[]}
-                  methodology={narrative.introductionMethodology}
-                />
-              )}
-            </NarrativeIntroContainer>
-          ),
-        },
-        ...narrative.sections.map((section) => {
-          // all sections except the conclusion should look like this
-          if ("chartData" in section) {
-            return {
-              title: section.title,
+    <Wrapper>
+      <ModelHydrator model={narrative}>
```

Comment:
Why this approach rather than using `withMetricHydrator`?

Reply:
literally just because the prop is not named `metric` .... that HOC was just so I didn't have to type `<ModelHydrator model={metric}>` like 8 times